### PR TITLE
add overly

### PR DESCRIPTION
**Pull request by CJ-Wright**
Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vend other packages
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by CJ-Wright** [comment]
@conda-forge/staged-recipes it seems that conda-build is building these in the wrong order?

**Comment by hmaarrfk** [comment]
Try rebasing.

**Comment by CJ-Wright** [comment]
@hmaarrfk I don't think that fixed this

**Comment by hmaarrfk** [comment]
You need to depend on sansio dash multiparty. Not underscore

;)

**Comment by CJ-Wright** [comment]
:sheep:

File: recipes/overly/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - h11
+    - python >=3.6
+    - sansio_multipart
```
Comment (CJ-Wright) [inline recipes/overly/meta.yaml]:
```suggestion
    - sansio-multipart
```

**Comment by CJ-Wright** [comment]
@conda-forge/staged-recipes both these are noarch and so only need to pass on linux

**Comment by hmaarrfk** [comment]
isn't OSX typically supposed to pass?

**Comment by npavlovikj** [comment]
> isn't OSX typically supposed to pass?

`noarch` recipes are only built on Linux, so osx and win are ok to fail.

**Comment by hmaarrfk** [comment]
I get that they are OK to fail, but why would they fall in this case?
All dependencies are installed, and the package should be ok to install right?

Typically they fail at the dependency stage, or at the "exe check" stage on windows.

**Comment by npavlovikj** [comment]
> I get that they are OK to fail, but why would they fall in this case?
> All dependencies are installed, and the package should be ok to install right?
> 
> Typically they fail at the dependency stage, or at the "exe check" stage on windows.

Right, but `noarch` packages are not built on OSX and Windows at all - one of the advantages of using `noarch` is saving CI usage, and using only one platform instead of the 3, https://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages.

**Comment by chrisburr** [comment]
> I get that they are OK to fail, but why would they fall in this case?
> All dependencies are installed, and the package should be ok to install right?

It's rare but sometimes packages inherently don't support macOS. In this case it's because `POLLRDHUP` doesn't exist on macOS (though it's not documented).

**Pull request merged**